pandas/core/groupby.py

```diff
@@ -191,6 +191,63 @@
     dtype: int64
     """)
 
+_numeric_operations_doc_template = """
+    Compute %(f)s of group values.
```

Comment:
We could maybe make a dict with "full" names (max -> the maximum, prod -> the product, ..), to give the first sentence a nicer read

Reply:
Also wondering, would "of each group" be clearer than "of group values" ?